src/hooks/use-play-card-audio.ts

```diff
@@ -63,14 +69,30 @@ export function usePlayCardAudio() {
     setActiveCardSide('front');
     await playCardAudio(frontAudio);
 
+    // Start capturing speech
+    const capturedSpeechPromise = captureSpeech(lessonCard.card.back.language as any);
```

Comment:
if a card has a language of `Default`, it needs to inherit from the parent

couple ways to do this:
- hold a reference to the deck in card/lesson-card
- hold the deck's language in the card/lesson-card

Reply:
Lol, forgot to do that. I should really leave some todo's lying around. 

I ended up restricting the language type when the lesson card is created